tools/slims.py

```diff
@@ -167,8 +167,10 @@ def more_fastqs(sample_name, run_tag):
         d['fastq paths'] = more_fq_paths
         #print(f'FQ PATHS:{more_fq_paths} ')
         for f_path in more_fq_paths:
+        # Only links if link doesn't already exist
+            if not os.path.islink(os.path.join(f"/home/xshang/ws_testoutput/symlinks/", os.path.basename(f_path))):
```

Comment:
I see spooky hardcoded paths to home

Reply:
oops, I added new crappy testing stuff, not ready for review yet